posts/2021/02/accessibility-whos-responsible.md

```diff
@@ -0,0 +1,130 @@
+support/documentation, to any other interaction. A comprehensive team who thinks to ask questions like, 
+“how would I use this if I could only use my keyboard?” or “would I be able to get the same information if 
+I were colorblind?” are starting to hold themselves and their team accountable. Taking responsibility is 
+key to starting and sustaining change.
+
+### Misconceptions
+
+Here are a few common concerns I’ve heard when people tell me why they can’t or haven’t worked on 
+accessibility. I’m going to paraphrase some replies I've heard when asking about accessibility in many 
+different environments (not only JupyterLab) over the years.
+
+**I don’t know anything!**
+And that’s fine. You don’t have to be an expert! Fortunately, there are already a lot of resources out 
+on the wide open internet, some even focused on beginners (some of my personal favorites are at 
+[The A11y Project](https://www.a11yproject.com/resources) and 
+[MDN](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility)). Of course, 
+it’s important to remember that learning will mean that you are likely to make mistakes and need to keep 
+iterating. This isn’t a one-and-done deal. And if you do have access to an expert, spending time to have 
+a foundation means they can help you tackle greater obstacles instead of just giving you the basics.
+
+**I don’t have time for another project!** 
+Accessibility doesn’t have to be your only focus. JupyterLab sure isn’t the only project I am working on, 
+and it won’t be in the near future. Any progress is better than no progress, and several people doing even 
+a little work can add up faster than you might think. Besides, there’s a good chance you won’t even have 
+to go out of your way to start improving accessibility. Start by asking questions about a project you are 
+already working on. Is there a recommended way to design and build a component? Is information represented 
+in more than one way? Is everything labeled?  It’s good practice and more sustainable to consider 
+accessibility as a regular part of your process instead of a special side project.
+
+**It’s not a good use of my energy to work on something that only affects a few people!**
+It’s not just a few people. Read what [WHO](https://www.who.int/en/news-room/fact-sheets/detail/disability-and-health) 
+and the [CDC](https://www.cdc.gov/ncbddd/disabilityandhealth/infographic-disability-impacts-all.html) have 
+to say about the number of people with disabilities.
+
+**I don’t want to make abled people’s experience different than it already is!**
+Depending on what you are doing, the changes might not be active or noticeable unless assistive technologies 
+or accessibility features are being actively used. And in many cases, accessibility features improve the 
+experience for all users and not just those they were designed for. Even if you aren’t convinced, I’d encourage 
```

Comment:
A great resource to link is anything related to the cut curb effect (helps to bring this idea to something more relatable to our day-to-day lives)